libs/wingsdk/src/std/map.ts

```diff
@@ -72,29 +82,38 @@ export class ImmutableMap {
 export class MutableMap extends ImmutableMap {
   /**
    * Removes all elements
+   *
+   * @macro ((map) => { for(const k in map){delete map[k]}; })($self$)
    */
   public clear(): void {
-    throw new Error("Abstract");
+    throw new Error("Macro");
   }
 
   /**
    * Removes the specified element from a map.
+   *
+   * @macro (delete $self$[$args$])
+   *
    * @param key The key
    * @returns true if the element was in the map
```

Comment:
the return value of the `delete` operator is different, it'll return true even if the property doesn't exist. In fact, it may even throw in some cases if we start using `strict` mode.

Maybe this to make it more wishy-washy?

```suggestion
   * @returns true if the given key is no longer present
```

Not ideal, but idk how much wrapper code we want to make for this usecase.